<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardView</class>
 <widget class="QWidget" name="StandardView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/bg/calendar.gif</normaloff>:/bg/calendar.gif</iconset>
  </property>
  <widget class="Sframe" name="frame" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{
	background-image: url(:/bg/scenary.jpg);
	border-radius: 40px;
	 border-style: groove;
     border-width: 4px;
     border-color:rgb(203, 203, 203); 

   
	
}</string>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>811</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="SLineEdit" name="lineEdit" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>701</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your commands here</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit 
{
	border-radius: 10px;
	border-style: ridge;
    border-width: 4px;
    border-color: rgb(84, 84, 84); 
	
	
	background-color: rgb(170, 85, 0);
}
</string>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>4</y>
       <width>151</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AR JULIAN</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_2
{
	
}</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'AR JULIAN'; font-size:16pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MV BOLI'; font-size:10pt; font-weight:400;&quot;&gt;Sunday 30-09-12&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MV BOLI'; font-size:14pt;&quot;&gt;16:40&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>93</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_2
{

}</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MV Boli'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffaa00;&quot;&gt;Available Commands: add, delete, mark, unmark, edit and find&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>731</width>
       <height>381</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabBar::tab:first {
background-color: qlineargradient(spread:pad, x1:0.528, y1:1, x2:0.534091, y2:0, stop:0 rgba(255, 111, 111, 255), stop:0.369318 rgba(225, 0, 0, 255), stop:0.636364 rgba(255, 0, 0, 255), stop:0.977273 rgba(255, 95, 95, 255));

border: 4px; 
border-color: rgb(152, 0, 0);
border-style:outset;  
border-top-left-radius: 30px;
border-top-right-radius: 10px;
min-width: 364px;
min-height: 30px;
}

QTabBar::tab:first:selected, QTabBar::tab:first:hover {
	background-color: qlineargradient(spread:pad, x1:0.528, y1:1, x2:0.534091, y2:0, stop:0 rgba(255, 104, 104, 255), stop:0.369318 rgba(225, 22, 22, 255), stop:0.636364 rgba(255, 23, 23, 255), stop:0.977273 rgba(255, 99, 99, 255));
}

QTabBar::tab:last {
	
	background-color: qlineargradient(spread:pad, x1:0.528, y1:1, x2:0.534091, y2:0, stop:0 rgba(109, 191, 64, 255), stop:0.369318 rgba(15, 155, 13, 255), stop:0.636364 rgba(12, 151, 22, 255), stop:0.977273 rgba(104, 192, 57, 255));
border: 4px; 
border-color:  rgb(0, 85, 0);
border-style:outset;  
border-top-left-radius: 30px;
border-top-right-radius: 10px;
min-width: 364px;
min-height: 30px;
}

QTabBar::tab:last:selected, QTabBar::tab:last:hover {
	background-color: qlineargradient(spread:pad, x1:0.528, y1:1, x2:0.534091, y2:0, stop:0 rgba(132, 191, 100, 255), stop:0.369318 rgba(32, 155, 30, 255), stop:0.636364 rgba(34, 151, 42, 255), stop:0.977273 rgba(137, 192, 107, 255));
}

QTabBar::tab:!selected {
margin-top: 10px; /* make non-selected tabs look smaller */
}
QTabBar::tab:first:selected {
margin-left: 0px; 
margin-right: -15px;
}

QTabBar::tab:last:selected {
margin-left: -15px; 
margin-right: 0px;
}

QTabBar::tab:only-one {
margin: 0; /* if there is only one tab, we don't want overlapping margins */
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>17</height>
      </size>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Event for the day</string>
      </attribute>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>728</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_3
{
	background-color: rgb(255, 170, 0);
	 border-style: ridge;
     border-width: 4px;
     border-color:  rgb(255, 0, 0);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>391</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#555500;&quot;&gt;Our greatest glory is not in never falling,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#555500;&quot;&gt; but in getting up every time we do.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Command Results</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>1</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Palatino Linotype'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#aa0000;&quot;&gt;S/N&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>319</x>
         <y>1</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Palatino Linotype'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#aa0000;&quot;&gt;RESULTS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>30</y>
         <width>721</width>
         <height>297</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#tableWidget
{
	
	
	background-color: rgb(222, 255, 190);

}</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>350</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>25</number>
       </attribute>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>728</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_4
{
	background-color: rgb(170, 255, 127);
	 border-style: ridge;
     border-width: 4px;
     border-color: rgb(170, 170, 0);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>391</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <zorder>frame_4</zorder>
      <zorder>label_5</zorder>
      <zorder>label_6</zorder>
      <zorder>tableWidget</zorder>
      <zorder>label_8</zorder>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-4</y>
      <width>821</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_4
{
	background-image: url(:/bg/testtitle.gif);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_3
{
	background-image: url(:/bg/small.gif);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2
{
	border-style: solid;
	border-radius: 5px;
	border-width: 3px;
	border-color: gray;
}
#pushButton_2:hover 
{
	border-style: outset;
	border-radius: 5px;
	border-width:3px;
	border-color: gray;
}

#pushButton_2:pressed 
{
	border-style: inset;
	border-radius: 5px;
	border-width:3px;
	border-color: gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/bg/toSmall.gif</normaloff>:/bg/toSmall.gif</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>757</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton
{
	border-style: solid;
	border-radius: 5px;
	border-width: 3px;
	border-color: gray;
}
#pushButton:hover 
{
	border-style: outset;
	border-radius: 5px;
	border-width:3px;
	border-color: gray;
}

#pushButton:pressed 
{
	border-style: inset;
	border-radius: 5px;
	border-width:3px;
	border-color: gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/bg/exit.gif</normaloff>:/bg/exit.gif</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SLineEdit</class>
   <extends>QWidget</extends>
   <header>slineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Sframe</class>
   <extends>QWidget</extends>
   <header>sframe.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>StandardView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
